app/logical/sources/strategies/deviant_art.rb

```diff
@@ -1,25 +1,36 @@
 # Asset URLs:
 #
-# * http://orig12.deviantart.net/9b69/f/2017/023/7/c/illustration___tokyo_encount_oei__by_melisaongmiqin-dawi58s.png
-# * http://pre15.deviantart.net/81de/th/pre/f/2015/063/5/f/inha_by_inhaestudios-d8kfzm5.jpg
-# * http://th00.deviantart.net/fs71/PRE/f/2014/065/3/b/goruto_by_xyelkiltrox-d797tit.png
+## NORMALIZED
```

Comment:
Sorry, but I find this a bit confusing so let me just check I understand it correctly: You divide these into "will be normalized", "will not be normalized but can be normalized" and "cannot be normalized" which you call "NORMALIZED", "NOT NORMALIZED", and "NOT NORMALIZABLE"

* "normalized" implies to me that it's already normalized
* "not normalized" implies to me that, well, it's not normalized as written, nothing more

Also, since you're already adding some cases why not handle those two under "not normalized" as well, should be fairly easy, shouldn't it? I would think even the `/deviation/{id}` would be better if extracting title and artist is too fiddly.

Reply:
I didn't want to make the deviantart regexes too confusing and rework them too much for this pull request, but honestly you're right, since I already added several cases I might as well fix those too and normalize all the normalizable. I'll add it tomorrow when I rebase everything.

Reply:
I think I'll add a separate variable for those though, otherwise it'll just become too hard to read and unmantainable.